dotnet-client-libraries/FluentEmail/FluentEmailExample/Controllers/EmailController.cs

```diff
@@ -0,0 +1,124 @@
+﻿using FluentEmailExample.Model;
+using FluentEmailExample.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace FluentEmailExample.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EmailController : ControllerBase
+    {
+        private readonly IEmailService _emailService;
+
+        public EmailController(IEmailService emailService)
+        {
+            _emailService = emailService ??
+                throw new ArgumentNullException(nameof(emailService));
+        }
+
+        [HttpGet("singleemail")]
+        public async Task<IActionResult> SendSingleEmail()
+        {
+            EmailMetadata emailMetadata = new("john.doe@gmail.com"
+                , "FluentEmail Test email"
```

Comment:
The commas should come after the previous parameter, and not on a newline. So for example:
```csharp
EmailMetadata emailMetadata = new("john.doe@gmail.com",
    "FluentEmail Test Email",
    "This is a test email from FluentEmail");